source/index.html.md

```diff
@@ -945,7 +941,7 @@ To protect against timing attacks, use a constant-time string comparison to comp
 
 # Changelog
 We take backwards compatibility seriously. The following list contains backwards compatible changes:
-
```

Comment:
I think you need the empty line or the formatting comes out weird.

Reply:
Yeah you're right... Don't know how that happened because this file gets auto-generated 🤷